### PR TITLE
Changed folder dir

### PR DESCRIPTION
**Pull request by wwyc**
As discussed I've updated the folder structure to combine the api and react_app.  

cd CourseHub
npm start 

.env file should be in CourseHub folder now

open browser:  http://localhost:9000/homepage

### CODE REVIEW

**Comment by illeanz** [comment]
Idk if it's just me, but I keep getting MongoNetworkError failed to connect :(

**Comment by wwyc** [comment]
> Idk if it's just me, but I keep getting MongoNetworkError failed to connect :(

Do you have a .env file set up inside your CourseHub folder?  [CourseHub/.env]
@illeanz

File: react_app/src/components/App/App.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import ContactPage from "../ContactUs/ContactUs";
 import AboutPage from "../AboutPage/AboutPage";
 import LoginPage from "../LoginPage/LoginPage";
 import {
-    BrowserRouter as Router,
+    HashRouter as Router,
```
Comment (wwyc) [inline react_app/src/components/App/App.js]: see BrowserRouter vs HashRouter:
https://stackoverflow.com/questions/51974369/hashrouter-vs-browserrouter

**Comment by illeanz** [comment]
Oh wow yeah you're right I'm stupid...
I cloned your repo and forgot to add the env file

File: routes/courses.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+var express = require('express');
```
Comment (wwyc) [inline routes/courses.js:1]: I didn't add this, it was just moved from api folder up to courseHub folder

File: react_app/src/components/App/App.js (hunk 1 above)
Comment (aaronNgu) [reply react_app/src/components/App/App.js]: what was the reason for the switch? couldn't quite infer from the article.

**Review by aaronNgu — APPROVED** [review]

**Review by aaronNgu — APPROVED** [review]

**Comment by wwyc** [comment]
met with TA today to discuss a few things - will make some updates according to his suggestions

**Comment by wwyc** [comment]
The TA suggested using api fallback so I removed the use of HashRouter.  Let me know what you think!

**Review by aaronNgu — APPROVED** [review]

**Pull request merged**